webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -1,3 +1,23 @@
+from flask import render_template
+from pydantic import ValidationError, root_validator
+from flask_babel import lazy_gettext as _l, ngettext, _
+from wtforms.validators import InputRequired
+from wtforms import SelectField
+from flask_wtf.csrf import generate_csrf
+
+from app.model.components import PauschbetragPersonProps
+from app.model.components.helpers import form_fields_dict
+from app.model.form_data import DisabilityModel, show_person_b
+from app.forms import SteuerlotseBaseForm
+from app.forms.steps.step import SectionLink
+from app.forms.steps.lotse.lotse_step import LotseFormSteuerlotseStep
+from app.forms.steps.lotse.personal_data import StepPersonAHasDisability, StepPersonBHasDisability, ShowPersonBPrecondition, StepFamilienstand
+
+def get_number_of_users(input_data):
+    if show_person_b(input_data):
+        return 2
+    return 1
```

Comment:
I think we have this quite some times in the code? Why copy this again?

Reply:
Replaced :grin: